### ISSUE TITLE
Any chance we could add descriptions to these packages?

### ISSUE THREAD

**Issue by levitanong** [janet-lang/pkgs#27]
While The Good Place puns are really entertaining, I can imagine newcomers would have a hard time finding what package they need from this package list. Adding a package description somehow would really aid in the usability and approachability of the ecosystem.

It would likely be a breaking change though, because it would require a change in data structure. One workaround might be to have a separate map, mapping package name to description, but it would be a funny lookup problem "okay, this is sounds like the package I need, now to look at the other map to see the URL." But that could be mitigated by a website similar to npmjs.org (jpm.org, anyone?)

**Comment by pepe** [comment]
Hello. Thank you for the wake-up call :-). I actually have a prototype for the page with the pkgs gallery at good-place.org (code https://git.sr.ht/~pepe/good-place/) that clone all the pkgs repositories and create the static page from the information in each project.janet file.

I have some more content for Good Place, like a blog, which would not be needed for the official one. Or at least at the beginning. But I would love to have posts by the authors of the packages and stuff like that under the pkgs.janet-lang.org or similar.

All I need is the community and mainly @bakpakin's confirmation that it all makes sense.

**Comment by uvtc** [comment]
@levitanong , descriptions are in each individual project's project.janet file. I wrote a quick script to go grab project.janet files and from that generate a directory page from it ([gen-janet-pkg-listing-page](https://github.com/uvtc/gen-janet-pkg-listing-page)), and others have created the [Powered By Janet site](https://goto-engineering.github.io/powered-by-janet/).

**Comment by levitanong** [comment]
@uvtc Cool! Though it's a static page generator, right? If that's the case, then wouldn't it become out of date unless there's a listener on this repo, rebuilding every time a new commit is added?

**Comment by uvtc** [comment]
@levitanong , yeah, mine is just a rough script that is not smart about updates. Not sure what the PBJ (Powered-by-Janet) site does.
